src/components/CampusAudioTour.tsx

```
@@ -0,0 +1,201 @@
+
+    // 1. Initialize High-Accuracy Browser Geolocation Tracker
+    const watchId = navigator.geolocation.watchPosition(
+      (position) => {
+        const { latitude, longitude } = position.coords;
+        setCoords({ lat: latitude, lon: longitude });
+        evaluateProximityMetrics(longitude, latitude);
+      },
+      (err) => {
+        setErrorText(`GPS error: ${err.message}. Enable accurate location tracking.`);
+      },
+      { enableHighAccuracy: true, timeout: 10000, maximumAge: 0 }
```

ESLint

[error] Insert `,`

(prettier/prettier)